.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           name: lint
           command: |
             ./scripts/lint-download.sh
-            make lint-check-diff
+            ./scripts/lint-check-diff.sh
```

Comment:
Could you please keep `make lint-check-diff` here

Reply:
I think we should not keep `make lint-check-diff` there, because the 
1) the script runs golangci lint with memory limitation. We can't decide for user what the memory config he should to use. 
2) the bash script has attempts. Attempts is not need for the user.

@artem-belov let me know if I missed something.